replication/li_wand_2016/main.py

```diff
@@ -29,14 +28,13 @@ def figure_6(args):
             f"Replicating the {position} half of figure 6 " f"with {params} parameters"
         )
 
-        # https://github.com/pmeier/CNNMRF/blob/fddcf4d01e2a6ce201059d8bc38597f74a09ba3f/run_trans.lua#L65-L70
+        # https://github.com/pmeier/CNNMRF/blob/fddcf4d01e2a6ce201059d8bc38597f74a09ba3f/run_trans.lua#L67
         style_loss_kwargs = {
-            "target_transforms": MRFOperator.scale_and_rotate_transforms(
-                num_scale_steps=1,
-                scale_step_width=5e-2,
-                num_rotate_steps=1,
-                rotate_step_width=7.5,
-            )
+            "patch_size": 2,
```

Comment:
Should this not be `patch_size=3`? This should correspond to the 9 entry in line [L67](https://github.com/pmeier/CNNMRF/blob/fddcf4d01e2a6ce201059d8bc38597f74a09ba3f/run_trans.lua#L67).

Reply:
Two things:

1. My comment referenced the wrong line. [L66]() is the relevant one.
2. Regardless of that, my local copy had these changes and I just used this. Good catch!